tests/translator/output/aws-us-gov/api_with_swagger_authorizer_none.json

```diff
@@ -1,499 +0,0 @@
-{
```

Comment:
Why we deleted this test file?

Reply:
Because this doesn't have a input template, it's never tested